### ISSUE TITLE
Alternate output format, inverse so that format is an array of objects

### ISSUE THREAD

**Issue by prettydiff** [Unibeautify/parse-framework#30]
Current the output is a single format of an object of arrays.

Example:

```javascript
{
    begin: [],
    lexer: [],
    lines: [],
    presv: [],
    stack: [],
    token: [],
    types: []
}
```

This format exists for two reasons:

* Arrays are faster to access than object trees.
* Parallel arrays are easier and less costly to work with when maintaining and extending the lexer files.

An easier format to work with, without sacrificing speed of access, would be an array of objects.  Simple, an inverse of the standard output format.

Example:

```javascript
[
    {
        begin: -1,
        lexer: "markup",
        lines: 0,
        presv: false,
        stack: "global",
        token: "<html>",
        types: "start"
    }
]
```

I will need to create a standard option to allow for this alternate output format.

**Comment by prettydiff** [comment]
I have this in the v 1.1.0 release, but it looks like I utterly failed to write documentation.  Keeping the issue open until I include the appropriate documentation.

There is an option to regulate the format of the output.  The option is named `outputFormat` and accepts the values `arrays` and `objects`.  The default is `arrays` which produces an object of the 7 arrays.  The value `objects` produces an array where each index is an object of 7 properties representing a single parsed token.

It can be run from the command line as

```
node js/runtimes/nodetest.js "js/runtimes/global.js" --raw --outputFormat
```

**Comment by prettydiff** [comment]
Resolved in 1.1.1 branch.
